pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -809,75 +813,105 @@ void MainEmuFrame::Menu_Capture_Screenshot_Screenshot_Click(wxCommandEvent & eve
 #ifndef DISABLE_RECORDING
 void MainEmuFrame::Menu_Recording_New_Click(wxCommandEvent &event)
 {
+	const bool notInitiallyPaused = !g_RecordingControls.IsEmulationAndRecordingPaused();
+	if (notInitiallyPaused)
+	{
+		g_RecordingControls.Pause();
+	}
 	NewRecordingFrame* NewRecordingFrame = wxGetApp().GetNewRecordingFramePtr();
 	if (NewRecordingFrame)
 	{
 		if (NewRecordingFrame->ShowModal() == wxID_CANCEL)
 		{
+			if (notInitiallyPaused)
+			{
+				g_RecordingControls.Unpause();
+			}
 			return;
 		}
-		// From Current Frame
-		if (NewRecordingFrame->GetFrom() == 0)
+		if (!g_InputRecording.Create(NewRecordingFrame->GetFile(), !NewRecordingFrame->GetFrom(), NewRecordingFrame->GetAuthor()))
 		{
-			if (!CoreThread.IsOpen())
+			if (notInitiallyPaused)
 			{
-				recordingConLog(L"[REC]: Game is not open, aborting new input recording.\n");
-				return;
+				g_RecordingControls.Unpause();
 			}
-			g_InputRecording.Stop();
-			g_InputRecording.Create(NewRecordingFrame->GetFile(), true, NewRecordingFrame->GetAuthor());
-		}
-		// From Power-On
-		else if (NewRecordingFrame->GetFrom() == 1)
-		{
-			g_InputRecording.Stop();
-			g_InputRecording.Create(NewRecordingFrame->GetFile(), false, NewRecordingFrame->GetAuthor());
+			return;
 		}
 	}
 	m_menuRecording.FindChildItem(MenuId_Recording_New)->Enable(false);
 	m_menuRecording.FindChildItem(MenuId_Recording_Stop)->Enable(true);
+	m_menuRecording.FindChildItem(MenuId_Recording_Restart)->Enable(true);
 }
 
 void MainEmuFrame::Menu_Recording_Play_Click(wxCommandEvent &event)
 {
+	const bool noTASLoaded = g_InputRecording.GetModeState() == INPUT_RECORDING_MODE_NONE;
```

Comment:
I've been trying to avoid the `TAS` terminology if possible as I feel like it's jargon and people are always asking "what is TAS".  I'd prefer this to be `noRecordingLoaded` for that reason.